src/components/object-info/ObjectInfo.js

```diff
@@ -71,7 +72,7 @@ const ObjectInfo = ({ className, type, cid, localPath, size, data, links, format
         ) : null}
         {format === 'unixfs' && data.type && ['directory', 'file'].some(x => x === data.type) ? (
           <a className='link avenir ml2 pa2 fw5 f6 blue' href={`https://ipfs.io/ipfs/${cid}`} target='_external'>
-            View on IPFS Gateway
+            {t('')}
```

Comment:
what is happening here :thinking:  ?

Reply:
MISSING KEY IS MISSING! @lidel @hacdias should I add the `en` value to the translation files or can this be automagically added later by a `tx pull`?

Reply:
It will be added automatically next time you do `tx pull -a`